packages/audits/src/components/AuditsTableCell.tsx

```diff
@@ -34,7 +34,7 @@ const Parser = new UAParser(undefined, { browser: ownRegexp });
 
 const UserAgent: CellComponent = ({ value }: { value: string }) => {
   const browser = useMemo(() => {
-    const browser = Parser.setUA(value).getBrowser().name;
+    const browser = value ? Parser.setUA(value).getBrowser().name : '';
```

Comment:
can you please check the implementation of getBrowser if has nullable return?

Reply:
The `getBrowser` function can't return null because this function [inits](https://github.com/faisalman/ua-parser-js/blob/master/src/ua-parser.js#L760) the response as undefined values.